docs/cli-usage.md

```diff
@@ -0,0 +1,96 @@
+## Using Kubernetes from the command line
```

Comment:
We should also add command-line flags for a curated set of labels, such as `--app=foo`, `--tier={fe,cache,be,db}`, `--uservice=redis`, `--env={dev,test,prod}`, `--stage={canary,final}, `--track={hourly,daily,weekly}`, `--release=0.4.3c2`. Exact ones TBD. We could allow arbitrary values -- the keys are important.  The actual label keys would be namespaced with `kubectl.kubernetes.io/`.